lib/src/server_messages.dart

```diff
@@ -270,3 +283,11 @@ class ErrorField {
     }
   }
 }
+
+class Notification {
+  final int processId;
```

Comment:
I believe that when a connection is made a process ID is assigned to it (this should be stored in Connection._processID). When NOTIFY is delivered, the process ID is included in the delivery to each subscriber that matches the process ID of the connection that triggered the NOTIFY. An interested subscriber may want to be able to filter out NOTIFYs that they themselves sent.